apps/nextjs-website/src/lib/strapi/__tests__/tutorials.test.ts

```diff
@@ -0,0 +1,321 @@
+import * as E from 'fp-ts/lib/Either';
+import { StrapiTutorialsCodec } from '@/lib/strapi/tutorial';
+
+const baseTutorialJson = {
+  id: 1,
+  attributes: {
+    title: 'title',
```

Comment:
You could replace these texts and use the functions in `mock-content.helper.ts`

Reply:
mock content helper it is an helper designed to be used in stories, here in tests i think is better to use raw json of http call